modules/autocomplete/main/index.coffee

```diff
@@ -63,15 +63,9 @@ findTerm = (term, forceMatch) ->
             data = data.sort hx.sort.compare
         data
 
+    groupedActive = new hx.Map hx.groupBy remainingResults, (i) -> not i.disabled
 
-    remainingResults = heading.concat(
-      remainingResults.sort (a, b) ->
-        if not a.disabled and b.disabled then -1
-        else if a.disabled and not b.disabled then 1
-        else hx.sort.compare(a.full, b.full)
```

Comment:
I think `.full` was something used in preferences for selecting locales - it should never have been in this file...